scripts/start.sh

```diff
@@ -1,9 +1,12 @@
-#!/usr/bin/env bash
+#!/bin/bash
```

Comment:
While alpine likely will always have bash at `/bin/bash`, `/usr/bin/env bash` is the more portable form.